liquibase-core/src/main/java/liquibase/change/core/SetColumnRemarksChange.java

```diff
@@ -17,6 +17,7 @@ public class SetColumnRemarksChange extends AbstractChange {
     private String columnName;
     private String remarks;
     private String columnDataType;
+    private boolean isView = false;
```

Comment:
Adding a not null/defaulted field to the change type like this is going to modify the definition used to generate checksums. So I think this will break everyone using setColumnRemark currently.

I think you have to leave it as Boolean and leave it null by default.

I also wonder if instead of a more specific isView with a boolean it should be an enumeration like `ownerType` or `relationType` that currently can have values of TABLE or VIEW but later on could be COLLECTION or whatever else different databases may want to add columns to?

Reply:
This makes me think that we should have tests around the checksum generation code so that we can be sure we don't accidentally break it going forward with changes like this.

Reply:
Yes, very much we do, in an automatically complete way. What we have now is too manual and scattered and one-off

Reply:
I logged a tech debt story: https://datical.atlassian.net/browse/DAT-12752